setup.cfg

```diff
@@ -88,9 +88,25 @@ addopts =
 # Why do we exclude this file from coverage?
 # Because coverage is not calculated correctly for pytest plugins.
 # And we completely test it anyway.
+branch = True
```

Comment:
We already have that here: https://github.com/wemake-services/django-test-migrations/blob/686d03a6e4898db8b6e49177b82deda916976f45/setup.cfg#L82-L83

Reply:
ahh, sorry I didn't notice usage of this flags.
However I think it's more intuitive to set all `coverage` related settings in `setup.cfg` `coverage` section and use `--cov*`  only if something cannot be configured by using `coverage` configuration (like for instance `--cov` or `--cov-report`).

I know `pytest-cov` has something like below in its docs: 

> Note that this plugin controls some options and setting the option in the config file will have no effect. These include specifying source to be measured (source option) and all data file handling (data_file and parallel options).

But in most cases `--cov*` flags will override `coverage` settings only if they are actually used.